admin/module/catalog-admin-module-sources/src/main/webapp/js/view/ModalSource.view.js

```diff
@@ -236,6 +236,8 @@ function (ich,Marionette,Backbone,ConfigurationEdit,Service,Utils,wreqr,_,$,moda
             if (!_.isUndefined(config)) {
                 var properties =  config.get('properties');
                 properties.set('shortname', name);
+                properties.set('id', name);
```

Comment:
If you pass an object into the set method, you can set multiple properties at the same time.    

````javascript
properties.set({
       'shortname': name,
       'id': name
});
````

Other than that, :+1: